src/Symfony/Component/Cache/Adapter/Client/MemcachedClient.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter\Client;
+
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+/**
+ * @author Rob Frawley 2nd <rmf@src.run>
+ */
+class MemcachedClient
+{
+    private static $serverDefaults = array(
```

Comment:
These are the distribution defaults for an installed memcached server with **no user configuration applied**. Additionally, this doesn't actually **add a default server** if no DSN is passed. At a minimum, the user still needs to pass `memcached:` as the DSN. An empty array or empty string will **not** add a server.